envs/samtools/samtools-1.9.yaml

```diff
@@ -1,28 +1,37 @@
 name: test-samtools
 channels:
-  - conda-forge
   - bioconda
+  - conda-forge
   - defaults
 dependencies:
   - _libgcc_mutex=0.1
   - _openmp_mutex=4.5
+  - bcftools=1.8
   - bzip2=1.0.8
-  - ca-certificates=2020.4.5.1
+  - ca-certificates=2020.12.8
+  - certifi=2020.12.5
   - curl=7.69.1
   - htslib=1.9
   - krb5=1.17.1
   - libcurl=7.69.1
   - libdeflate=1.5
   - libedit=3.1.20170329
+  - libffi=3.2.1
   - libgcc-ng=9.2.0
   - libgomp=9.2.0
   - libssh2=1.8.2
   - libstdcxx-ng=9.2.0
+  - lofreq=2.1.3.1
   - ncurses=6.1
-  - openssl=1.1.1f
+  - openssl=1.1.1i
+  - pip=20.3.3
+  - python=3.6.8
+  - readline=7.0
   - samtools=1.9
+  - setuptools=51.1.2
+  - sqlite=3.26.0
   - tk=8.6.10
+  - wheel=0.36.2
   - xz=5.2.5
   - zlib=1.2.11
-prefix: /home/bgrande/miniconda3/envs/test-samtools
-
+prefix: /home/lakshay/miniconda3/envs/test-samtools
```

Comment:
Did you mean to hard code this "lakshay" dependent path? Will this code break if you leave and your account is deactivated?

Reply:
Good point, need to ask someone

Reply:
she means your GSC account, I think